src/workflows/foraging.bonsai.layout

```diff
@@ -92,7 +92,7 @@
         <Y>3</Y>
       </Location>
       <Size>
-        <Width>240</Width>
+        <Width>330</Width>
```

Comment:
@hagikent is it possible to add screenshots of what the these changes look like? I can't really tell what the differences will look like

Reply:
Never mind you already provided screenshots my bad!